circle.yml

```diff
@@ -620,6 +625,27 @@ jobs:
           path: /tmp/artifacts
       - store-npm-logs
 
+  desktop-gui-visual-tests:
+    <<: *defaults
+    parallelism: 1
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          command: yarn lerna exec --scope @packages/desktop-gui "yarn build-prod"
+      - run:
+          name: Desktop GUI server
+          command: yarn start
+          working_directory: packages/desktop-gui
+          background: true
+      - run:
+          # will use PERCY_TOKEN environment variable if available
+          command: |
+            CYPRESS_KONFIG_ENV=production \
+            yarn lerna exec --scope @packages/desktop-gui "yarn percy exec -- yarn cypress:run --spec cypress/integration/settings_spec.js"
```

Comment:
for now we have visual tests in just a single spec file, in the future can extend and run several files